boxoffice/static/js/models/sidebar.js

```diff
@@ -0,0 +1,43 @@
+export const SideBarModel = {
+  items: function(id) {
+    let baseUrl = 'ic/' + id;
+    return [
+      {
+        url: baseUrl,
+        title: 'DashBoard',
+        icon: 'fa-dashboard',
+        view: 'dashboard'
+      },
+      {
+        url: baseUrl + '/items',
+        title: 'Items',
+        icon: 'fa-ticket',
+        view: 'items'
+      },
+      {
+        url: baseUrl + '/discounts',
+        title: 'Discounts',
+        icon: 'fa-tag',
+        view: 'discounts'
+      },
+      {
+        url: baseUrl + '/orders',
+        title: 'Orders',
+        icon: 'fa-shopping-cart',
+        view: 'orders'
+      },
+      {
+        url: baseUrl + '/assignees',
+        title: 'Assignees',
+        icon: 'fa-users',
+        view: 'assignees'
+      },
+      {
+        url: baseUrl + '/reports',
+        title: 'Reports',
+        icon: 'fa-area-chart',
+        view: 'report'      
+      }
+    ]
+  }
+};
```

Comment:
New line at the end of the file. Also, I think it's better to only have the routes that are functional as of now. We can add the rest when they become active.